arangod/Replication2/ReplicatedState/ReplicatedState.tpp

```diff
@@ -384,6 +396,8 @@ void LeaderStateManager<S>::recoverEntries() {
         if (auto self = weak.lock(); self != nullptr) {
           auto guard = self->_guardedData.getLockedGuard();
           guard->_leaderState->onRecoveryCompleted();
+          guard->_recoveryCompleted = true;
+          LOG_CTX("1b3d0", DEBUG, self->_loggerContext) << "recovery completed";
```

Comment:
This log ID is the same as the previous one.

Reply:
correct. But did jenkins not fail with that error?

Reply:
`arangodb-ANY-cppcheck-pr` failed, but there is only `sourceInfo.log` in there. I suppose it was the log ids, it's not obvious where to look for the failure in this case.

Reply:
`arangodb-ANY-check-guidelines` is the one that checks log ids.

Reply:
ah, I know why. This is `LOG_CTX`. And the bash script used to check for log ids does not recognize this. Because it is (of course) a different implementation and does not use `utils/checkLogIds.py`